_maps/shuttles/shiptest/whiteship_kilo.dmm

```diff
@@ -1040,251 +1002,226 @@
-	dir = 8
-	},
-/obj/machinery/atmospherics/components/unary/shuttle/heater{
-	dir = 8
-	},
-/obj/machinery/door/window/westright,
-/turf/open/floor/plating/airless,
-/area/ship/engineering/engine)
-"qw" = (
-/obj/effect/turf_decal/industrial/warning{
-	dir = 4
-	},
-/obj/machinery/power/shuttle/engine/electric{
+/turf/open/floor/mineral/titanium/blue,
+/area/ship/bridge)
+"ng" = (
+/obj/effect/decal/cleanable/greenglow,
+/obj/machinery/power/terminal{
 	dir = 8
 	},
 /obj/structure/cable{
-	icon_state = "0-8"
+	icon_state = "0-4"
 	},
-/obj/structure/window/reinforced{
+/obj/machinery/atmospherics/pipe/simple/orange/visible{
+	dir = 6
+	},
+/obj/effect/turf_decal/industrial/warning{
 	dir = 8
 	},
-/turf/open/floor/plating/airless,
+/turf/open/floor/plating,
 /area/ship/engineering/engine)
-"rc" = (
-/obj/effect/turf_decal/industrial/warning/corner{
-	dir = 4
+"nJ" = (
+/obj/effect/turf_decal/industrial/warning,
+/obj/effect/decal/cleanable/blood/old,
+/obj/structure/cable{
+	icon_state = "1-2"
 	},
-/obj/effect/decal/cleanable/dirt,
-/obj/effect/decal/cleanable/greenglow,
 /obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
-	dir = 4
+	dir = 6
 	},
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
-	dir = 8
+	dir = 10
 	},
-/turf/open/floor/mineral/titanium/blue,
-/area/ship/bridge)
-"rs" = (
-/obj/machinery/porta_turret/centcom_shuttle/weak{
-	dir = 4;
-	name = "Old Mining Turret"
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/turf/open/floor/plating,
+/area/ship/cargo)
+"nO" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/machinery/conveyor{
+	dir = 1;
+	id = "NTMSLoad";
+	name = "off ramp"
+	},
+/obj/machinery/door/firedoor/border_only,
+/obj/structure/plasticflaps,
+/turf/open/floor/plating,
+/area/ship/cargo)
+"oj" = (
+/obj/machinery/atmospherics/pipe/simple/orange/visible{
+	dir = 9
 	},
-/turf/closed/wall/mineral/titanium,
-/area/ship/crew)
-"rX" = (
 /obj/effect/turf_decal/industrial/warning{
 	dir = 10
 	},
+/turf/open/floor/plating,
+/area/ship/engineering/engine)
+"oP" = (
 /obj/effect/decal/cleanable/blood/old,
 /obj/effect/decal/cleanable/dirt,
 /obj/machinery/atmospherics/components/unary/tank/toxins{
 	dir = 1
 	},
+/obj/effect/turf_decal/industrial/warning{
+	dir = 6
+	},
 /turf/open/floor/plating,
 /area/ship/engineering/engine)
-"sl" = (
+"pV" = (
 /obj/effect/turf_decal/industrial/warning{
-	dir = 4
-	},
-/obj/effect/turf_decal/industrial/warning/corner{
-	dir = 8
+	dir = 1
 	},
 /obj/effect/decal/cleanable/blood/old,
-/obj/effect/decal/cleanable/cobweb,
-/obj/structure/cable{
-	icon_state = "0-2"
-	},
-/obj/machinery/shower{
-	pixel_y = 18
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4,
+/mob/living/simple_animal/hostile/netherworld/migo{
+	environment_smash = 0;
+	faction = list("neutral");
+	melee_damage_lower = 5;
+	melee_damage_upper = 10;
+	name = "Maurice"
```

Comment:
```suggestion
	name = "maurice"
```
Please keep his name lowercase, this was an intentional design choice when I added him

Reply:
Alrighty.